packages/mds-types/index.ts

```diff
@@ -105,7 +105,7 @@ export interface TripEntry {
   min_violation_dist: number | null
   avg_violation_dist: number | null
   events: TripEvent[]
-  telemetry: TripTelemetry[][]
+  telemetry: { [event: number]: TripTelemetry[] }
```

Comment:
Just for clarity, perhaps `telemetry: { [timestamp: number]: TripTelemetry[] }` as it appears the object timestamps as the property name/index. This type declaration occurs in multiple places so perhaps a type alias would be appropriate, `TripEntryTelemetry` or somesuch.